plugin/ToolBar.qml

```diff
@@ -22,49 +22,39 @@ import Sailfish.Silica 1.0
 PanelBackground {
     id: toolbar
 
-    property var flickable: undefined
-    property int parentHeight
-    property bool hidden
+    property Item flickable
+    property bool forceHidden
     property bool autoHide: true
+    property bool trackMovement: true
```

Comment:
Yes, this is redundant. Introduced because I couldn't come up with a single word that would cover both cases. Hmm, stateChangesEnabled?